Components/build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+plugins {
+    kotlin("multiplatform")
+    id("jacoco")
+    id("maven-publish")
+    id("com.android.library")
+}
+
+apply(from = "../gradle/component.gradle")
+
+group = "com.splendo.mpp"
+version = "0.0.2"
```

Comment:
Shouldn't it be imported from `gradle.library_version`

Reply:
yes it should